pippo-core/src/main/java/ro/fortsoft/pippo/core/Response.java

```diff
@@ -62,12 +62,14 @@ public String getContentType() {
     }
 
     public Response contentType(String contentType) {
+        checkCommitted();
```

Comment:
Yes.  But I'd like to make it clear that attempting to modify the response after commit() is a no-no.  I think that is better than thinking you can, failing silently, and then debugging/scratching your head.  Of course that is an opinion, not truth.